src/lib/gssapi/mechglue/g_initialize.c

```diff
@@ -467,11 +467,13 @@ load_if_changed(const char *pathname, time_t last, time_t *highest)
 	time_t mtime;
 
 	mtime = check_link_mtime(pathname, &mtime);
+        if (mtime == (time_t)-1 && last == (time_t)-1)
+            mtime = 0;
 	if (mtime == (time_t)-1)
 		return;
 	if (mtime > *highest)
```

Comment:
I think this also needs a special case for `*highest == (time_t)-1`, assuming we are guarding against unsigned time_t at all.

Reply:
I think you are right, fixing